coordinator/tests/services/validation/bundle-validator.spec.ts

```diff
@@ -186,6 +186,28 @@ describe("MedicationRequest consistency checks", () => {
 
     validateValidationErrors(validationErrors)
   })
+
+  test("Should reject message where 2 or more medication requests share an identifier", () => {
+    const identifier = [
```

Comment:
can define type like `const identifier: Array<fhir.Identifier> = ` rather than forcing it at the end